bin/helg

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env node
-var helg = require('../');
-console.log(helg.ere());
```

Comment:
Ah, sorry about that. The main motive was to give a nicer output in the cli. I'll refactor! :neckbeard: